.storybook/preview.js

```diff
@@ -17,7 +17,7 @@ export const parameters = {
   docs: {
     source: {
       excludeDecorators: true,
-      type: 'source',
```

Comment:
hmm, but we don't use `dynamic` as did FAI, is `source` alias for dynamic ?

Reply:
![image](https://github.com/user-attachments/assets/098466d1-678a-422f-b54d-0e5137b1c6aa)
source seems like it doesn't exist(?) the docs only mention `auto, dynamic, code`, so not sure 😆.

One thing to note though is that only a few of our stories actually take advantage of the args and our public docsite doesn't even let you modify those args. 

> dynamic: Renders the story source with dynamically updated arg values

to me dynamic is actually not even worth it at that point especially if we don't even use it since we end up just having a static code block.

Reply:
right 😃

so this whole storybook DX is confusing indeed (starting with confusing documentation...). I double checked if `code` prop has any value and it does not.... 

to add to this confusion, those docs are for `Source` component which is not being properly used in our case it seems ( dunno why ). 

it is what it is I guess ... 🥹

Reply:
hahaha yup at least we can fix that horrible infinte loop :'D